### DISCUSSION TITLE
Create a custom folder under ./api/{domain}/

### DISCUSSION THREAD

**Discussion by pigman8857** [strapi/strapi#8213] [Legacy Topics]
Just a question,

Is it ok to create folder under ./api/{domain}/, for example, ./api/{domain}/helpers or ./api/{domain}/utils 
 not sure if there is a problem going on after that.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
